OsmAnd-java/src/main/java/net/osmand/router/MissingMapsCalculator.java

```diff
@@ -58,7 +59,8 @@ public MissingMapsCalculator(OsmandRegions osmandRegions) {
 	}
 
 
-	public boolean checkIfThereAreMissingMaps(RoutingContext ctx, LatLon start, List<LatLon> targets, boolean checkHHEditions)
+	public boolean checkIfThereAreMissingMaps(RoutingContext ctx, LatLon start, List<LatLon> targets,
```

Comment:
flag calledByCalcOnline is not requested

Reply:
The problem is the method checkIfThereAreMissingMaps() is re-used and original targets[] is overwritten by online result when the "Calculate online" button is pressed. calledByCalcOnline (and its followers passed between classes) are required to detect the fact that the Missing Maps fragment is called after calc online resulted - and finally to hide Calc Online button against double-request.